src/cmd/common/utils.go

```diff
@@ -35,3 +37,14 @@ func ExitOnInterrupt() {
 		}
 	}()
 }
+
+// NewClusterOrDie creates a new Cluster instance and waits for the cluster to be ready or throws a fatal error.
+func NewClusterOrDie() *cluster.Cluster {
```

Comment:
Is there a reason why we wouldn't want to pass the `ctx` we already have from `cmd` in here?

Reply:
Good catch. Updated to pass in a `ctx`